examples-next/basic/.keystone/schema-types.ts

```diff
@@ -26,10 +26,6 @@ export type UserWhereInput = {
   readonly OR?: ReadonlyArray<UserWhereInput | null> | null;
```

Comment:
I have no idea why these schema-types.ts changes are here. 
looking at the .gitignore for this example, not ignoring schema-types seems to be intentional. 

IF someone could fill me in about this, that would be great

Reply:
The short version is "this is fine, this file will be removed soon".